### ISSUE TITLE
Can't load message: Invalid `href`

### ISSUE THREAD

**Issue by ShadowJonathan** [vector-im/element-web#21430]
### Steps to reproduce

Event source;

```json5
{
  "content": {
    "body": "I could just make an alias like #matrix:non-matrix.org and host a phising Element at element.non-matrix.org. Any server can add aliases to a room or appear similar enough, so not sure that is a good idea.",
    "format": "org.matrix.custom.html",
    "formatted_body": "I could just make an alias like #<a href=\"matrix:non-matrix.org\">matrix:non-matrix.org</a> and host a phising Element at element.non-matrix.org. Any server can add aliases to a room or appear similar enough, so not sure that is a good idea.",
    "msgtype": "m.text"
  },
  "type": "m.room.message",
  "unsigned": {},
  // Some event keys redacted
}
```

I believe the culprit is `#<a href=\"matrix:non-matrix.org\">matrix:non-matrix.org</a>`

#### What did you expect?

The message to maybe display with a fallback, or mangled up a bit.

#### What happened instead?

![image](https://user-images.githubusercontent.com/22740616/158550499-693f7db6-f7cd-4f1f-bc75-092e9644789f.png)

### Operating system

PopOS 21.10

### Application version

Element version: 1.10.6 Olm version: 3.2.8

### How did you install the app?

Flatpak

### Homeserver

jboi.nl

### Will you send logs?

No

**Comment by dbkr** [comment]
No logs, so will need to try & reproduce with the given message.

**Comment by t3chguy** [comment]
This got fixed in 1.10.7 I believe.

![image](https://user-images.githubusercontent.com/2403652/159864595-c1b594e8-34d2-457c-a5dc-55d78f46f0df.png)
